cucumber/authenticators_jwt/features/authn_jwt_mapping_claims.feature

```diff
@@ -0,0 +1,61 @@
+Feature: JWT Authenticator - Mapping claims sanity
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: jwks-uri
+
+      - !variable
+        id: token-app-property
+
+      - !group hosts
+
+      - !permit
+        role: !group hosts
+        privilege: [ read, authenticate ]
+        resource: !webservice
+    """
+    And I am the super-user
+    And I successfully set authn-jwt jwks-uri variable with value of "myJWKs.json" endpoint
+    And I successfully set authn-jwt "token-app-property" variable to value "user"
+
+  Scenario: ONYX-10473 Mapping claims with subsequent annotation
+    Given I extend the policy with:
+    """
+    - !variable conjur/authn-jwt/raw/mapping-claims
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+        authn-jwt/raw/branch: main
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/hosts
+      member: !host myapp
+    """
+    And I add the secret value "branch:ref" to the resource "cucumber:variable:conjur/authn-jwt/raw/mapping-claims"
+
+    And I issue a JWT token:
+    """
+    {
+      "user":"myapp",
```

Comment:
in examples and annotations just use word valid and invalid to make it easier

Reply:
Discussed it
Each field should have its own value.
For example `valid-project-id`.
Using the same value like `valid` for each fields can hide some problems